test/transportSpec.js

```diff
@@ -140,6 +140,13 @@ describe('Transport', function () {
       assert.equal(details.headers['header'], 'value')
     })
 
+    it('should throw an error if header value contains a line feed', function () {
```

Comment:
I would also add tests to make sure that you can include both URLs and Unicode characters (e.g. for names)